modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptNestjsClientCodegen.java

```diff
@@ -49,7 +49,7 @@
     public static final String STRING_ENUMS = "stringEnums";
     public static final String STRING_ENUMS_DESC = "Generate string enums instead of objects for enum values.";
 
-    protected String nestVersion = "6.0.0";
+    protected String nestVersion = "8.3.1";
```

Comment:
We should default this to 8.4.0, I ran into some issues installing fastify plugins on 8.3.x